src/handlers/misc/Leaderboard.ts

```diff
@@ -29,6 +47,34 @@ export default class LeaderboardHandler extends AbstractHandler {
             result += `${toPush}\n`
         }
 
+
         return msg.channel.send(result)
     }
+
+    private hasExpired() : boolean
+    {
+        const date = new Date();
+
+        try{
```

Comment:
Why is this in a `try`?

Reply:
Just in case something goes wrong with Date comparison, the program shouldn't just terminate. I figured it would be better to just return false and continue with the older data